runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingWorld.java

```diff
@@ -272,7 +278,8 @@ public boolean isInBounds(WorldPoint loc)
 			new FarmingPatch("North", Varbits.FARMING_4771, PatchImplementation.ALLOTMENT),
 			new FarmingPatch("South", Varbits.FARMING_4772, PatchImplementation.ALLOTMENT),
 			new FarmingPatch("", Varbits.FARMING_4773, PatchImplementation.FLOWER),
-			new FarmingPatch("", Varbits.FARMING_4775, PatchImplementation.CRYSTAL_TREE)
+			new FarmingPatch("", Varbits.FARMING_4775, PatchImplementation.CRYSTAL_TREE),
+			new FarmingPatch("", Varbits.FARMING_4774, PatchImplementation.COMPOST) // TODO: Find correct varbit
```

Comment:
@abextm did you check this varbit? It might be made up.

Reply:
I forgot to remove the TODO, the prif bin tracks correctly at least in my testing.